### ISSUE TITLE
"Update imports" fails

### ISSUE THREAD

**Issue by scarlac** [unlight/sublime-import-helper#8]
Plugin does nothing when activated or claims there are no matching imports. Seems related to this error in the console:
```
import_root: /Users/seph/jobs/*****
source_folders: ['/Users/seph/jobs/*****']
Run command: get_packages
Exception in thread Thread-9:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "./python3.3/threading.py", line 858, in run
  File "utils in /Users/seph/Library/Application Support/Sublime Text 3/Installed Packages/ImportHelper.sublime-package", line 27, in run_command
  File "utils in /Users/seph/Library/Application Support/Sublime Text 3/Installed Packages/ImportHelper.sublime-package", line 76, in exec
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1448, in _execute_child
NotADirectoryError: [Errno 20] Not a directory
```

Using Sublime Text 3, Stable Channel, Build 3126, on macOS 10.12.2.

**Comment by mreq** [comment]
Same on Ubuntu 16.04, build 3126

**Comment by unlight** [comment]
It is hard to say what is wrong here.
Need additional information about folder structure.
I've released version 1.0.6 which may not fail for unexisting directory and can give more info about error.

**Comment by unlight** [comment]
Will be closed, if no feedback will be provided.

**Comment by mreq** [comment]
The error stays the same.

```
Run command: get_packages
Exception in thread Thread-72:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "./python3.3/threading.py", line 858, in run
  File "utils in /home/petr/.config/sublime-text-3/Installed Packages/ImportHelper.sublime-package", line 27, in run_command
  File "utils in /home/petr/.config/sublime-text-3/Installed Packages/ImportHelper.sublime-package", line 76, in exec
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1448, in _execute_child
NotADirectoryError: [Errno 20] Not a directory
```

**Comment by Ledzz** [comment]
+1, got a same issue.

Last line says `Folder name is invalid`

Folder root is `D:\sites\technokratos\seiko`, it's a typical Angular2 app.

```Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "./python3.3/threading.py", line 858, in run
  File "utils in C:\Users\Artem\AppData\Roaming\Sublime Text 3\Installed Packages\ImportHelper.sublime-package", line 27, in run_command
  File "utils in C:\Users\Artem\AppData\Roaming\Sublime Text 3\Installed Packages\ImportHelper.sublime-package", line 74, in exec
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1110, in _execute_child
NotADirectoryError: [WinError 267] Неверно задано имя папки```

**Comment by unlight** [comment]
@scarlac @mreq @Ledzz
Finally figure out the root of issue :zomba:
Issue happens because package files were packed by PackageControl to `ImportHelper.sublime-package` archive.
To fix the issue:
1. Close Sublime
2. Remove `Data/Installed Packages/ImportHelper.sublime-package`
3. Start Sublime

PackageControl should install missing package.

Check that ImportHelper was installed to `Data/Packages/ImportHelper`

**Comment by mreq** [comment]
Sorry, now it fails on

```
Import Helper
TypeError: Cannot read property 'dependencies' of undefined
    at readPkgUp.then.p (/path/to/packages/Packages/ImportHelper/backend_run.js:13:36)
```

**Comment by unlight** [comment]
@mreq 
It should be fixed since version 1.0.10 (already deployed to Package Control)

**Comment by scarlac** [comment]
@mreq Can you confirm that it's been fixed? Then we can close the issue.
